ITR/data/base_providers.py

```diff
@@ -265,17 +300,20 @@ def _convert_projections_to_series(self, company: ICompanyData, feature: str,
                 )
             else:
                 projections = company_dict[feature][list(projection_scopes.keys())[0]]['projections']
-        return pd.Series(
-            {p['year']: p['value'] for p in projections},
-            name=company.company_id, dtype=f'pint[{emissions_units}/({production_units})]')
```

Comment:
shouldn't we return series from here?

Reply:
This is code that's blocked by a breakpoint.  The logic is not finished.  We need to build test cases that land here and we need to implement from there.

Reply:
as we are in a production code, should this conditional branch be protected by exception, rather then a breakpoint?